src/daemon/daemon.cpp

```diff
@@ -159,23 +159,27 @@ auto make_cloud_init_vendor_config(const mp::SSHKeyProvider& key_provider,
     config["ssh_authorized_keys"].push_back(ssh_key_line);
     config["timezone"] = request->time_zone();
     config["system_info"]["default_user"]["name"] = username;
-    config["packages"].push_back("pollinate");
 
-    auto pollinate_user_agent_string =
-        fmt::format("multipass/version/{} # written by Multipass\n", multipass::version_string);
-    pollinate_user_agent_string +=
-        fmt::format("multipass/driver/{} # written by Multipass\n", backend_version_string);
-    pollinate_user_agent_string += fmt::format("multipass/host/{} # written by Multipass\n",
-                                               multipass::platform::host_version());
-    pollinate_user_agent_string += fmt::format("multipass/alias/{}{} # written by Multipass\n",
-                                               !remote_name.empty() ? remote_name + ":" : "",
-                                               pollinate_alias);
+    if (request->image() != "fedora")
```

Comment:
Add a comment here about why, for future readers? Just looking at the code, I'm not sure what makes Fedora a special case, and whether I'd need to add another distro here in the future.

(In the long term, there's probably some abstract rule that makes sense here, but one step at a time...)

Reply:
The pollinate server, which is hosted by Canonical, is also scheduled for deprecation in the near future.